docs/config/validate-links.json

```diff
@@ -2,5 +2,6 @@
   "ignorePatterns": [
     { "pattern": "^https://mvnrepository\\.com" },
     { "pattern": "^http://127.0.0.1:8080" }
```

Comment:
```suggestion
    { "pattern": "^http://127.0.0.1:8080" },
```

Reply:
:) thanks
I tried to push but was too late